### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by leonceaklin** [webstudio-is/webstudio-community#84] [Wishlist]
Most of my customers need their page to be localized in different languages. That's why I wanted to ask how you all feel about integrating first-party localization features into Webstudio.

**Comment by kof** [comment]
We want that ofc, that's not a question if we want, the question is only when :)

**Comment by sedubois** [comment]
A big +1. Any chance to add it to the [roadmap](https://github.com/orgs/webstudio-is/projects/11/views/1)? Multi-language is quite an essential feature for us.

**Reply by kof** [reply]
CMS will already do this and most CMSs we are looking into have localization options
